.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "contracts/lib/optimism"]
 	path = contracts/lib/optimism
 	url = https://github.com/ethereum-optimism/optimism
+[submodule "contracts/lib/interop-lib"]
```

Comment:
adding a submodule increases repository management complexity. do you think we could add this as a Foundry dependency instead? there's probably a good reason not to but wanted to raise it as an option to see what you think!

Reply:
actually I take it back. since interop-lib probably won't be stable for a while as changes are being made submodule gives us better control.

Reply:
oh interesting, is there a way to install foundry dependencies without using sub modules? [`forge install`'s](https://book.getfoundry.sh/reference/forge/forge-install) default behavior is to add the dependency as a submodule and I think the only other option outside of submodules is to use node_modules, but definitely open to other options if it's possible!

Reply:
Yeah you can ignore me. :)